grafana-plugin/src/containers/RouteLabelsDisplay/RouteLabelsDisplay.tsx

```diff
@@ -32,28 +32,15 @@ export const RouteLabelsDisplay: React.FC<RouteLabelsDisplayProps> = ({ labels,
   };
 
   const onLoadKeys = async (search?: string) => {
-    let result = undefined;
-
-    try {
-      result = await labelsStore.loadKeys(search);
-    } catch (error) {
-      openErrorNotification('There was an error processing your request. Please try again');
-    }
-
+    const result = await labelsStore.loadKeys(search);
     return splitToGroups(result);
   };
 
   const onLoadValuesForKey = async (key: string, search?: string) => {
-    let result = undefined;
+    if (!key) {return [];}
```

Comment:
looks like wrong formatting

Reply:
Yeah, `yarn run lint:fix` seems not to complain on this one... I don't really understand why `pre-commit` yields different results compared to `lint:fix`, they look the same